Update Elasticsearch schema for 4.6 (#2341)
ibexa/documentation-developer: docs/update_and_migration/from_4.5/update_from_4.5.md

                  - identifier: activity_log_owner
                    values: []
```

## Update Elasticsearch schema

> Elasticsearch index must be erased, the schema updated, and the index rebuild.

Reviewer: ```suggestion
When you make changes to the configuration files, you need to erase Elasticsearch index, update the schema, and rebuild the index.
```

Author: ```suggestion
Elasticsearch schema changes. You need to erase its index, update the schema, and rebuild the index.
```

> Elasticsearch schema evolves. You need to erase its index, update the schema, and rebuild the index.

Reviewer 2: ```suggestion
Elasticsearch schema evolves.
You need to erase its index, update the schema, and rebuild the index.
```
How does the schema evolve, because of what?
With time, because of changes to the the content model?
One or two reasons for schema's evolution would help get into the mind of the developer who needs to perform these actions.

Author: @dabrt 
```suggestion
Elasticsearch schema's templates change with the addition of new features such as spellchaking.
You need to erase the index, update the schema, and rebuild the index.
```

> To delete the index, you can use an HTTP request. Use the command as in the following example:

Reviewer 2: ```suggestion
To delete the index, you can use an HTTP request.
Use the command as in the following example:
```

```bash

> curl --request DELETE 'http://elasticsearch:9200/_all'

Reviewer 3: ```suggestion
curl --request DELETE 'https://elasticsearch:9200/_all'
```
I know this is just an example, but maybe we should use https in examples as well?

```

> To update the schema then reindex, use the following commands:

Author: ```suggestion
To update the schema, and then reindex the content, use the following commands:
```
